tooling-tests/src/test/resources/test-src/input/tool_test_init_1/main.bal

```diff
@@ -0,0 +1,14 @@
+import ballerina/io;
+configurable int port = ?;
+configurable string host = ?;
+configurable string user = ?;
+configurable string database = ?;
+configurable string password = ?;
+
+public function main() {
+    io:println(port == 3306);
+    io:println(user == "root");
+    io:println(host == "localhost");
+    io:println(password == "");
+    io:println(database == "");
+}
```

Comment:
```suggestion
import ballerina/io;

configurable int port = ?;
configurable string host = ?;
configurable string user = ?;
configurable string database = ?;
configurable string password = ?;

public function main() {
    io:println(port);
    io:println(user);
    io:println(host);
    io:println(password);
    io:println(database);
}
```

Reply:
Update other test resources as well.

Reply:
Updated in [181e367](https://github.com/ballerina-platform/persist-tools/pull/9/commits/181e36711c0bbd5cde7ab0dd0fce2b1445b72e65)